templates/portuguese/README.html

```diff
@@ -43,87 +43,85 @@
-<dd>Permite cargar una partida haciendo clic en uno de los espacios de guardado. Haciendo clic en "Auto" se accede a los espacios de guardado automático</dd> 
-<dt>Opciones</dt> 
-<dd>Cambia las opciones del juego:
-<dl> 
-<dt>Pantalla</dt> 
-<dd>Alterna entre la pantalla completa o el modo ventana.</dd> 
-<dt>Transiciones</dt> 
-<dd>Controla la presentación de transiciones entre las pantallas del juego.</dd> 
-<dt>Veloc. texto</dt> 
-<dd>Controla la velocidad con la que aparece el texto. Cuanto más a la derecha se selecciona esta barra, el texto se presentará más deprisa. La posición total a la derecha presenta el texto instantáneamente..</dd> 
-<dt>Joystick</dt> 
-<dd>Permite el control del joystick.</dd> 
-<dt>Saltar</dt> 
-<dd>Alterna entre saltar los mensajes ya vistos (en cualquiera de las partidas) y saltar todos los mensajes.</dd> 
-<dt>Comenzar salto</dt> 
-<dd>Vuelve al juego y comienza a saltar los mensajes.</dd> 
-<dt>Tras opciones</dt> 
-<dd>Controla si el salto se detiene cuando se llega a un menú.</dd> 
-<dt>Veloc. auto-avance</dt> 
-<dd>Controla el avance automático. Cuanto más a la izquierda se selecciona esta barra, más breve es el tiempo entre los avances del juego. La posición total a la derecha anula el auto-avance.</dd> 
-<dt>Volumen de música, sonido y voz</dt> 
-<dd>Controla el volumen de los canales de la música, los efectos de sonido y la voz, respectivamente. Cuanto más a la derecha se seleccionan, mayor es el volumen.</dd> 
+<p><a name="Basic_Help" id="Basic_Help"></a></p>
+<h2><span class="editsection">[<a href="/w/index.php?title=renpy/Help&amp;action=edit&amp;section=1" title="Edit section: Basic Help">edit</a>]</span> <span class="mw-headline">Ajuda Básica</span></h2>
+<p>Para avançar no jogo, clique com o <tt>botão esquerdo</tt> do mouse ou pressione as teclas <tt>espaço</tt> ou <tt>enter</tt>. Nos menus, clique com o <tt>botão esquerdo</tt> para fazer uma escolha, ou use as teclas do cursor para selecionar uma opção e <tt>enter</tt> para ativá-la.</p>
+<p><a name="Game_Menu" id="Game_Menu"></a></p>
+<h3><span class="editsection">[<a href="/w/index.php?title=renpy/Help&amp;action=edit&amp;section=2" title="Edit section: Game Menu">edit</a>]</span> <span class="mw-headline">Menu do jogo</span></h3>
+<p>Durante o jogo, clique com o <tt>botão direito</tt> ou pressione a tecla <tt>ESC</tt> para entrar no menu do jogo. O menu do jogo oferece as seguintes opções:</p>
+<dl>
+<dt>Retornar</dt>
+<dd>Retorna para o jogo.</dd>
+<dt>Salvar</dt>
+<dd>Permite salvar o jogo clicando em um dos espaços de salvamento.</dd>
+<dt>Carregar</dt>
+<dd>Permite carregar uma partida clicando em um dos espaços de armazenamento. Clicando em "Auto" são usados os espaçõs de salvamento automático</dd>
+<dt>Opções</dt>
+<dd>Alterar as opções do jogo:
+<dl>
+<dt>Tela</dt>
+<dd>Alterna entre tela cheia ou modo janela.</dd>
+<dt>Transições</dt>
+<dd>Controla a exibição de transições entre as telas do jogo.</dd>
+<dt>Veloc. texto</dt>
```

Comment:
The game translation files use "Velocidade do texto", there's no need to use abbreviation